pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -363,7 +363,7 @@ type configWrapper struct {
 	configPath string
 	// path to system-wide registries.conf.d directory, or "" if not used
 	configDirPath string
-	// path to user specificed registries.conf.d directory, or "" if not used
+	// path to user specified registries.conf.d directory, or "" if not used
```

Comment:
(Shouldn’t this actually be “user-specified”? Either way it’s an improvement, and not adding extra changes that the tool did not suggest would make the PR better-contained.)

Reply:
I think we should leave it the as `user specified`